Sources/XCTestDynamicOverlay/XCTIsTesting.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+public let _XCTIsTesting: Bool = {
```

Comment:
I'm underscoring this for now since there's a small chance for false positives...dunno if we want something more robust before providing a public-sanctioned API.